examples/misc/lib/effective_dart/docs_good.dart

```diff
@@ -157,16 +159,23 @@ void miscDeclAnalyzedButNotTested() {
   // #enddocregion markdown
 }
 
+class DuplicateFlagException implements Exception {}
+
 class IOError {}
 
 class PermissionError {}
 
+class Query {}
+
+class QueryResult {}
+
 class Widget {}
 
 // #docregion redundant
 class RadioButtonWidget extends Widget {
-  /// Sets the tooltip to [lines], which should have been word wrapped using
-  /// the current font.
+  /// Sets the tooltip to [lines].
+  ///
+  /// The lines should be word wrapped using the current font.
```

Comment:
```suggestion
  /// Sets the tooltip to [lines].
  /// [lines] represents a list of strings and each string
  /// in this list represents a line of text in the tooltip.
  ///
  /// The lines should be word wrapped using the current font.
```

We really should be defining parameters, in my opinion. I have no idea what `lines` is supposed to represent. A chat for a later date. For now, I've added an update to hopefully capture what `lines` is for.